client/src/document/ingredients/prose.tsx

```diff
@@ -1,10 +1,21 @@
 import { DisplayH2, DisplayH3 } from "./utils";
 
-export function Prose({ section }) {
-  return <div dangerouslySetInnerHTML={{ __html: section.content }} />;
+export function Prose({ id, section }: { id?: string; section: any }) {
+  return (
+    <section
+      aria-describedby={id}
+      dangerouslySetInnerHTML={{ __html: section.content }}
+    />
+  );
 }
 
-export function ProseWithHeading({ id, section }) {
+export function ProseWithHeading({
```

Comment:
I wonder whether we couldn't just merge `Prose` and `ProseWithHeading` now that both accept the id parameter. 🤔

Reply:
I would be open to that.

Reply:
Done in 1ba27e1.